src/main/java/DatabaseInteractin/Product.java

```diff
@@ -0,0 +1,59 @@
+package DatabaseInteractin;
+public class Product {
```

Comment:
Instead of having this class, you can just use the one generated by protobuf that has all these fields

Reply:
The definition of the product was removed from the proto file. This class is used to make and manipulate lists.